mobile/native/src/api.rs

```diff
@@ -231,7 +230,6 @@ pub fn run(
     db::init_db(&app_dir, get_network())?;
     let runtime = ln_dlc::get_or_create_tokio_runtime()?;
     ln_dlc::run(app_dir, seed_dir, runtime)?;
-    event::subscribe(ChannelFeePaymentSubscriber::new());
```

Comment:
❓Why did we have to move this?

Reply:
because I needed the `channel_manager` when creating the `ChannelFeePaymentSubscriber`